sdks/java/core/src/main/java/org/apache/beam/sdk/testing/TestStream.java

```diff
@@ -142,7 +142,7 @@ private Builder(Coder<T> coder, ImmutableList<Event<T>> events, Instant currentW
      */
     public Builder<T> advanceWatermarkTo(Instant newWatermark) {
       checkArgument(
-          newWatermark.isAfter(currentWatermark), "The watermark must monotonically advance");
+          !newWatermark.isBefore(currentWatermark), "The watermark must monotonically advance");
```

Comment:
<!--new_thread; commit:f1bd748cd47c2a1bf5245e2d4913d043cf0d4925; resolved:0-->
This change is useful for getting quiescence without advancing the watermark, yes? I think it is worth an explicit mention in the javadoc and/or some syntactic sugar like `quiesce()`.